packages/ui-codemods/lib/__node_tests__/__testfixtures__/colors.input.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
```

Comment:
the test fixtures in this PR are revived from our old code, they don't need to be reviewed